app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -168,7 +169,9 @@ class FollowUsers extends Component {
                 </figure>
                 <div className="user-info">
                   <h4 className="user-name">{user.name}</h4>
-                  <p className="user-summary">{user.summary}</p>
+                  <p className="user-summary">
+                    {he.unescape(user.summary || '')}
```

Comment:
is there a chance that the summary, unescaped, can be used for XSS ?
I'm thinking this because the summary can be edited by the user and the onboarding can be accessed again after a user is already registered.

Reply:
I've tried a few things and it seems to be safe. Do you have any other suggestion for me to test out?
![image](https://user-images.githubusercontent.com/15793250/81071833-410eca80-8eb3-11ea-9d71-0660a13b6534.png)